### ISSUE TITLE
Parse URDF links and create forward kinematic model

### ISSUE THREAD

**Issue by dresio** [dresio/veritas#8]
This will enable more robust training as it will keep the gradients intact for backprop
